docs/en-US/config.yml.md

```diff
@@ -10,80 +10,185 @@ permalink: /docs/en-US/config/
 
 **Before VVV 3.2 the file was `vvv-config.yml` and `vvv-custom.yml` in the VVV root.**
 
-Here's the full default config file, with every key and option that VVV supports:
+Here's the full default config file:
 
 ```yaml
+---
+
```

Comment:
I was hoping to link to it rather than embed it, that way we don't have to keep going back and updating it.

Reply:
I'd noticed.... but we don't have any one file that actually lists _every_ option.

Reply:
more that the embed itself is problematic, people draw conclusions, it'd be better to expand the docs with the available options

Reply:
Ok, let's move away from it. I did actually add (at least most) of the missing commands, but I'm also sure I missed a few.